attest/tpm_windows.go

```diff
@@ -42,6 +42,12 @@ type windowsTPM struct {
 func (*windowsTPM) isTPMBase() {}
 
 func probeSystemTPMs() ([]probedTPM, error) {
+	// Initialize Tbs.dll here so that it's linked only when TPM support is required.
+	if tbs == nil {
+		tbs = windows.MustLoadDLL("Tbs.dll")
+	}
+	tbsGetDeviceInfo = tbs.MustFindProc("Tbsi_GetDeviceInfo")
```

Comment:
Move this into the if statement as well (its duplicated work for the procedure to be loaded twice - these globals should be treated as singletons)